src/components/TabNavigation.js

```diff
@@ -50,14 +52,34 @@ const TabNavigation = () => {
           </React.Fragment>
         }
       />
-      {enableIntegrations && (
-        <Tab
-          eventKey={INTEGRATIONS}
-          title={
-            <TabTitleText>{intl.formatMessage({ id: 'sources.integrations', defaultMessage: 'Integrations' })}</TabTitleText>
-          }
-        />
-      )}
+      {enableIntegrations &&
```

Comment:
Can we have a fallback to phase 1? If `enableIntegrations` is correct we'll show whole integrations table, if the `enableBreakdown` is true we'll show the list of tabs.

Reply:
Thank you, this needed an update, done.